_config.yml

```diff
@@ -57,7 +57,18 @@ conference:
 
   location:
     url: '/location'
-    map: true
+    map: 
+      enable: true
+      default_zoom: 17
+      home_coord: 47.37808, 8.53935
+      map_provider: "OpenStreetMap.Mapnik"
```

Comment:
I would remove all except the `enable`, `default_zoom` and `map_provider` properties - otherwise they are the default for every website using this template (if they do not explicitly overwrite the settings)

Reply:
I thought so as well but couldn't figure out where to store defaults. Should this directly go into the liquid markup in the js file? That feels ugly..

Reply:
What I meant is to write your coed in a way that it works even if the configuration is only partially complete. Everything you write in the `_config.yml` file of this theme will be the default for all websites using the theme (this is good for a parameter like `enable` but less the `home_coord`). Anyway, I wasn't aware of the Liquid `default:` pipeline - thanks for pointing that out - and think the way you did it now is fine.